### PR TITLE
Apply a filter on activity for dojo map search results

### PR DESCRIPTION
**Pull request by Wardormeur**
Optimatly, it should be done b-end side

### CODE REVIEW

**Comment by Wardormeur** [comment]
closes #1019

File: web/public/js/controllers/dojos-map-controller.js [hunk 1]
```diff
@@ -211,9 +209,18 @@ function cdDojosMapCtrl($scope, $window, $state, $stateParams, $translate, $geol
     $scope.markers = [];
   }
 
+  function filterDojos (dojos) {
```
Comment (DanielBrierton) [inline web/public/js/controllers/dojos-map-controller.js]: `filterInactiveDojos` would be more descriptive of what it actually does

File: web/public/js/controllers/dojos-map-controller.js [hunk 2]
```diff
@@ -211,9 +209,18 @@ function cdDojosMapCtrl($scope, $window, $state, $stateParams, $translate, $geol
...
+    var filteredDojos = [];
```
Comment (DanielBrierton) [inline web/public/js/controllers/dojos-map-controller.js]:
```javascript
return _.filter(dojos, function (dojo) {
  return dojo.stage !== 4;
});
```

Comment (Wardormeur) [inline web/public/js/controllers/dojos-map-controller.js]: Creppy pasta :>

**Pull request merged**